[RHACS] New version 3.67.2 Release Notes (#39924)
openshift/openshift-docs: release_notes/367-release-notes.adoc

* *ROX-8698*: In {product-title-short} 3.67.0, the TLS verification would fail when you integrated {product-title-short} with {ocp} OAuth server for {ocp} 4.8 and later. This issue has been fixed.

[id="resolved-in-version-3672"]
=== Resolved in version 3.67.2

Release date: December 16, 2021

* *ROX-8773*: Before this update, when integrating with Microsoft Teams, the {product-title-short} user interface field validation did not pass certain Microsoft Teams webhook addresses. This issue has been fixed.
* *ROX-8736*: In {product-title-short} 3.67.0, the `roxctl image check` command would retry on policy failures that broke builds. This issue has been fixed.

> * *ROX-8702*: In {product-title-short} 3.67.0, when using OpenShift OAuth, users' e-mails were displayed incorrectly as their username if an e-mail was unavailable. This issue has been fixed.

Reviewer: ```suggestion
* *ROX-8702*: In {product-title-short} 3.67.0, when using OpenShift OAuth, email display names incorrectly appeared as the username, if an email address was unavailable. This issue has been fixed.
```
I am not entirely sure if this suggestion represents the issue, but I do believe the sentence would benefit from a rewrite.

Reviewer 2: I've updated it to 
```
* *ROX-8702*: In {product-title-short} 3.67.0, when using OpenShift OAuth, the user name 
is incorrectly listed as the *User email*, if an email address was unavailable. 
This issue has been fixed.
```

> * *ROX-8702*: In {product-title-short} 3.67.0, when using OpenShift OAuth, the user name is incorrectly listed as the *User email*, if an email address was unavailable. This issue has been fixed.

Reviewer 3: Use present tense: "..., if the email address is unavailable..."
